apiserver/pkg/chat/chat_docs.go

```diff
@@ -210,13 +211,13 @@ func (cs *ChatServer) SummarizeConversationDoc(ctx context.Context, req Conversa
 		klog.V(5).Infof("Generate embeddings from file %s to vectorstore for conversation %s", doc.Filename, req.ConversationID)
 		errEmbedding = cs.GenerateSingleDocEmbeddings(ctx, req, doc, documents)
 		if errEmbedding != nil {
+			klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", "")
 			errStr += fmt.Sprintf(" ErrEmbedding: %s", errEmbedding.Error())
 			// break once error occurs
 			ctx.Done()
 			return
 		}
-		klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", "")
-		klog.V(5).Infof("Generate embeddings for doc %s is successful!")
+		klog.V(5).Infof("Generate embeddings for doc %s is successful!", doc.Filename)
```

Comment:
IMO, Should add req.ConversationID to the logs to make it easier to locate each request run for debugging purposes

Reply:
Done

Reply:
Done